src/Standards/PSR12/Docs/Properties/ConstantVisibilityStandard.xml

```diff
@@ -0,0 +1,25 @@
+<documentation title="Constant Visibility">
+    <standard>
+    <![CDATA[
+    Visibility must be declared on all class constants if your project PHP minimum version supports class constant visibilities (PHP 7.1 or later).
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Constant visibility declared.">
+        <![CDATA[
+class Foo
+{
+    <em>private</em> const BAR = 'bar';
+}
+        ]]>
+        </code>
+        <code title="Invalid: Constant visibility not declared.">
+        <![CDATA[
+class Foo
+{
+    <em></em>const BAR = 'bar';
```

Comment:
I wonder if the example would be clearer if the `em` tags included the `const` keyword (in both the valid + invalid code samples) ?

Reply:
Yeah, makes sense 👍🏼